--- OCTEONTX-SDK-old/dts/cn83xx-linux.dtsi	2019-12-26 09:17:21.818748066 -0500
+++ OCTEONTX-SDK/dts/cn83xx-linux.dtsi	2020-01-03 15:32:34.003710101 -0500
@@ -782,6 +782,19 @@
                                         <0 0 0 3 &gic0 0 0 0 30 4>, /* INTC */
                                         <0 0 0 4 &gic0 0 0 0 31 4>; /* INTD */
                 };
+
+		firewall-intr@0 {
+			compatible = "cavium,firewall";
+			interrupts = <0 15 1>,
+				     <0 16 1>,
+				     <0 17 1>,
+				     <0 18 1>,
+				     <0 19 1>,
+				     <0 20 1>,
+				     <0 21 1>,
+				     <0 22 1>;
+		};
+		
 	};
 
 	aliases {
